x/incentives/keeper/gauge.go

```diff
@@ -227,6 +230,9 @@ func (k Keeper) CreateGroup(ctx sdk.Context, coins sdk.Coins, numEpochPaidOver u
 		return 0, fmt.Errorf("Invalid splitting policy, needs to be Evenly got %s", splittingPolicy)
 	}
 
+	// TODO: remove gauge creation logic from here.
+	// Instead, call `CreateGauge` directly
+	// Update `CreateGauge` to be able to handle the group type.
```

Comment:
Note: tracked https://github.com/osmosis-labs/osmosis/issues/6513

Reply:
Had a comment on this but added directly to the issue so it doesn't get lost when we merge this PR

Reply:
Replied in the issue